src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -201,12 +201,40 @@ private static OpenApiResponses GetOpenApiResponses(MethodInfo method, EndpointM
                 responseContent[contentType] = new OpenApiMediaType();
             }
 
-            responses[statusCode] = new OpenApiResponse { Content = responseContent };
+            responses[statusCode] = new OpenApiResponse
+            {
+                Content = responseContent,
+                Description = GetResponseDescription(statusCode)
+            };
         }
-
         return responses;
     }
 
+    private static string GetResponseDescription(string statusCode)
+    {
+        if (statusCode.Length != 3)
+        {
+            return "";
+        }
+
+        char first = statusCode[0];
+        switch (first)
+        {
+            case '1':
+                return "information";
```

Comment:
```suggestion
                return "Information";
```

Nit: these status descriptions are often title-cased. Let's update here and elsewhere.